src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -69,6 +69,9 @@ public class RestoreCommand
         private const string IsLockFileValidForRestore = "IsLockFileValidForRestore";
         private const string LockFileEvaluationResult = "LockFileEvaluationResult";
 
+        // names for central package management version information
+        private const string CentralVersionManagementEnabled = "CentralVersionManagementEnabled";
```

Comment:
```suggestion
        private const string IsCentralVersionManagementEnabled = "IsCentralVersionManagementEnabled";
```

On par with IsLockFileEnabled and other booleans

Reply:
Updated.